### ISSUE TITLE
Tree view: "Nothing to show" after IDE start

### ISSUE THREAD

**Issue by OliverO2** [serras/fir-tree#26]
Versions: FIR Tree 0.4.0, IntelliJ IDEA 2024.2.3 (Ultimate Edition)

To reproduce:
1. Start the IDE.
2. Select the project view in the tool window bar 
3. Open a Kotlin source file in the editor.
4. Click the FIR Tree icon in the tool window bar.

"Nothing to show" will appear. Clicking "Refresh" does not help.

Workaround:
* With the FIR Tree window open: Switch to another file in the editor, then switch back to the Kotlin source file.

**Comment by serras** [comment]
I've tried to reproduce this bug, but I'm not able to. The plug-in requires the _Indexing_ phase to be finished (which itself requires the project to be loaded), and after that the tool window should refresh itself.

Does this issue still reproduce in 0.4.2?

**Comment by OliverO2** [comment]
Yes, it does reproduce: [Screen recording with FIR Tree 0.4.2, IntelliJ IDEA 2024.3 (Ultimate Edition)](https://github.com/user-attachments/assets/882d480a-dd7d-4354-afd0-559f33524e83)

**Comment by serras** [comment]
Thanks for the video. Unfortunately this does not reproduce in my system :( Just to try to figure out where to look at: 
- does the same happen if the file is not on the `test` set?
- does this happen in non-Multiplatform files?

**Comment by OliverO2** [comment]
It did reproduce
* with a file on the `main` set, and
* with a scratch file (which I'd consider non-KMP).

It did not reproduce
* when opened as a second project,
* when starting the IDE with the project in question, having FIR Tree view pre-selected (instead of project view).

I have attached my little test project and also a FIR plugin exception I got just now when opening my test project:
[kotlin-multiplatform-arena.zip](https://github.com/user-attachments/files/18414399/kotlin-multiplatform-arena.zip)
[FIR-tree-exception.zip](https://github.com/user-attachments/files/18414400/FIR-tree-exception.zip)
